rel/overlay/etc/vm.args

```diff
@@ -82,7 +82,8 @@
 ##      [{certfile, "</path/to/erlserver.pem>"},
 ##       {secure_renegotiate, true}]},
 ##     {client,
-##      [{secure_renegotiate, true}]}].
+##      [{secure_renegotiate, true},
+##       {verify, verify_none}]}].
```

Comment:
I'm very uncomfortable deliberating disabling security if this gets into the release tarball.

Reply:
I tried to use `{verify, verify_peer}`, but get errors about bad certificates, or just hang with `*** ERROR: Shell process terminated!`. So add `{verify, verify_none}` to suppress the warning.
```
*** ERROR: Shell process terminated! (^G to start new job) ***
=NOTICE REPORT==== 6-May-2023::13:58:20.360922 ===
TLS client: In state certify at ssl_handshake.erl:2080 generated CLIENT ALERT: Fatal - Bad Certificate
```